Bio/motifs/__init__.py

```diff
@@ -179,7 +176,8 @@ def __init__(self, instances=None, alphabet=None):
             if self.length is None:
                 self.length = len(instance)
             elif self.length != len(instance):
-                message = "All instances should have the same length (%d found, %d expected)" % (len(instance), self.length)
+                message = "All instances should have the same length (%d found, %d expected)" % (
+                len(instance), self.length)
```

Comment:
Can you revert this unrelated change (at some point we will be apply black style to this file anyway)